index.html

```diff
@@ -239,6 +239,51 @@ <h3 class="sk">
         </li>
       </ul>
     </section>
+    <section class="contact">
+      <div class="announcement">
+        <p>I'm alway interested in hearing about new projects, so if you'd like to chat please get in touch.</p>
+      </div>
+      <form action="#" method="POST" class="form">
+        <ul>
+          <li>
+            <input type="text" placeholder="Full name">
+          </li>
+          <li>
```

Comment:
- [ ] Please avoid using the `ul` inside the form. You can still use the form and with the help of `flexbox` you can easily attain the structure. As I noticed that you used the `flexbox` on your `ul` but you can optimize it more and make your code semantically more correct which makes your website more accessible. Whereas please note that `ul` and `li` are just meant for listing purposes so use it wisely 😄 . If you do the changes your code looks more leaner and easy to understand.